src/groups/GroupActions.js

```diff
@@ -167,7 +170,7 @@ export const makeQueryFromState = (/*type*/) => {
           commands: [
             ...matches,
             // {$sort: {'statistics.comments.all.all.count': -1}},
-            {$skip: 0},
+            {$skip: page * 20},
```

Comment:
We should make the page size dynamic.  I can see in some cases bringing back, say, 100 users being a smoother experience as the trigger to load more wouldn't come until more scrolling has been done.